docs/kss/starter-pages/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Techne Starter Pages</title>
+    <link rel="icon" type="image/png" href="images/favicon-96x96.png" sizes="96x96">
+    <link rel="stylesheet" href="../public/dist/css/techne.min.css">
+    <link rel="stylesheet" href="css/demo.css">
+</head>
+<body>
+    
+    <!-- top nav -->
+    <div class="top-nav navbar navbar-fixed-top">
```

Comment:
i don't think we should be using fixed elements in this "demo" wrapper.  There are a bunch of styles in demo.css that are in place to override the styles in place by the techne classes.